### PR TITLE
Update verification.py

### PR DESCRIPTION
**Pull request by TPPSellinger**
fixes the XivsHudson test

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Looks good to me @TPPSellinger. The checks are complaining about a minor formatting issue. See the [documentation](https://tskit.readthedocs.io/en/latest/development.html#pre-commit-checks) for more details on this (there's info on how to install pre-commit too).

File: verification.py [hunk 1]
```diff
@@ -1651,6 +1649,7 @@ def _run(self, xi_model, **kwargs):
             model_str = "hudson"
             if model != "hudson":
                 model_str = "Xi"
+                simulate_args["Ne"] = 2*(math.sqrt(int(simulate_args["Ne"])))
```
Comment (jeromekelleher) [inline verification.py:1652]: A comment here would be helpful I think. "We need to rescale Ne by blah because blah blah".

Comment (TPPSellinger) [reply verification.py:1654]: Okay. I hope the comment is clear.

Comment (jeromekelleher) [reply verification.py:1655]: Perfect. Just need to get the formatting fixed now and we're good. All you really need to do is run "black" on the code, like
```
python3 -m pip install black
python3 -m black verification.py
git commit -a --amend
```
and that should do it.

Comment (TPPSellinger) [reply verification.py]: Thanks ! Sorry  for all the trouble, as  I thought is was a small commit I went through the GitHub UI.

**Comment by TPPSellinger** [comment]
Everything is fine (or some squashing is expected) ?

**Comment by jeromekelleher** [comment]
It would be nice to squash them all right @TPPSellinger  - 6 commits is a bit more than we want for this change!

**Pull request closed without merging**